modules/stanford_events_views/stanford_events_views.views_default.inc

```diff
@@ -1367,6 +1393,15 @@ function stanford_events_views_views_default_views() {
   $handler->display->display_options['filters']['field_stanford_event_datetime_value']['field'] = 'field_stanford_event_datetime_value';
   $handler->display->display_options['filters']['field_stanford_event_datetime_value']['operator'] = '>=';
   $handler->display->display_options['filters']['field_stanford_event_datetime_value']['default_date'] = 'now';
+  /* Filter criterion: Content: Event Type (field_s_event_type) */
+  $handler->display->display_options['filters']['field_s_event_type_tid']['id'] = 'field_s_event_type_tid';
+  $handler->display->display_options['filters']['field_s_event_type_tid']['table'] = 'field_data_field_s_event_type';
+  $handler->display->display_options['filters']['field_s_event_type_tid']['field'] = 'field_s_event_type_tid';
+  $handler->display->display_options['filters']['field_s_event_type_tid']['value'] = array(
```

Comment:
We don't know that this tid is available for every site that uses this branch. Nor do we know if tid 29 is the term we want as they are inconsistent. We cannot have this set to a term id unless that term is bundled into this module. But we cannot bundle the term as that would be content. We will need to find a different way to accomplish the filter that does not require filtering by term id.